packages/nextjs/app/token-vendor/page.tsx

```diff
@@ -44,21 +53,8 @@ const TokenVendor: NextPage = () => {
 
   const { data: vendorContractBalance } = useBalance({
     address: vendorContractData?.address,
-    watch: true,
```

Comment:
we need the `watch` and `pending`, otherwise we fall into this issue (https://github.com/Quantum3-Labs/speedrunstark/issues/49)

Reply:
@Nadai2010 waiting for this to be fixed

Reply:
fixed